test/WebJobs.Script.Tests.Integration/EndToEndTestFixture.cs

```diff
@@ -143,7 +144,13 @@ protected virtual void CreateTestStorageEntities()
 
         public virtual void Dispose()
         {
-            Host.Stop();
+            try
+            {
+                Host.Stop();
+            }
+            catch
+            {
+            }
```

Comment:
The try catch was added because the service bus listener will throw on StopAsync. Submitted https://github.com/Azure/azure-webjobs-sdk/pull/1054 to address this.

Up to you guys if you want to wait on those changes getting in before merging this or not.